lib/features/apply/presenation/bloc/cubit/apply_cubit.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:bloc/bloc.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:jobs_tdd/features/apply/domain/entities/apply_info.dart';
+import 'package:jobs_tdd/features/apply/domain/useCases/apply_usecase.dart';
+
+import '../../../../../cores/locator.dart';
+
+part 'apply_state.dart';
+part 'apply_cubit.freezed.dart';
+
+class ApplyCubit extends Cubit<ApplyState> {
+  ApplyCubit() : super(ApplyState.initial());
+
+  Future<void> apply(ApplyInfo applyInfo) async {
+    // print(userSignUp.toModel());
+    await locator.get<ApplyUseCase>().call(applyInfo).then((value) {
```

Comment:
Why not pass it to this classes object and use it as a field? So that `get_it` can help with "injecting" it.